go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/decred/dcrd/rpc/jsonrpc/types/v3 v3.0.0
 	github.com/decred/dcrd/txscript/v4 v4.1.0
 	github.com/decred/dcrd/wire v1.6.0
-	github.com/decred/dcrdata/v7 v7.0.0-20211216152310-365c9dc820eb
+	github.com/decred/dcrdata/v7 v7.0.0
```

Comment:
Is this really tied to the latest master commit? Why don't you tie it to this commit: https://github.com/decred/dcrdata/commit/e7e8ab2402609b0094a5b0ba05fa0a3dce67f053

Reply:
no it's not, based on the issue https://github.com/crypto-power/cryptopower/issues/26 it's tied to the v1.8 commit

Reply:
My point is that the above dependency seems to be tied to the official release of v7.0.0 but the commit we want happened after that.

Reply:
Run this command to import changes as of this commit: https://github.com/decred/dcrdata/commit/fa4d8e1b4e8e2ea7752e5ce95c3c8e4a3beec6bd

```
 go get github.com/decred/dcrdata/v8@fa4d8e1b4e8e2ea7752e5ce95c3c8e4a3beec6bd
```

Reply:
that was the exact command i executed

Reply:
@dmigwi some packages still made reference to v8. they have been upgraded